mod/discussions/start.php

```diff
@@ -11,6 +11,12 @@
 function discussion_init() {
 
 	elgg_register_page_handler('discussion', 'discussion_page_handler');
+	
+	elgg_register_menu_item('site', [
```

Comment:
Please no. Unless a group enables a discussion, there are none to view and no way to create them.